Cannot find namespace 'Popper'.

### Bug description:

When I run `ng serve`, show this error:

```
ERROR in node_modules/ng2-semantic-ui/dist/misc/util/services/positioning.service.d.ts(35,21): error TS2503: Cannot find namespace 'Popper'.
node_modules/ng2-semantic-ui/dist/modules/sidebar/services/sidebar.service.d.ts(13,11): error TS2503: Cannot find namespace 'Popper'.
node_modules/ng2-semantic-ui/dist/modules/sidebar/services/sidebar.service.d.ts(14,12): error TS2503: Cannot find namespace 'Popper'.
node_modules/ng2-semantic-ui/dist/modules/sidebar/services/sidebar.service.d.ts(15,10): error TS2503: Cannot find namespace 'Popper'.
node_modules/ng2-semantic-ui/dist/modules/sidebar/services/sidebar.service.d.ts(16,13): error TS2503: Cannot find namespace 'Popper'.
```

### Version of Angular, ng2-semantic-ui, and Semantic UI:

NodeJS: 7.10.1

```
{
  "name": "izagro",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build --prod",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular-devkit/core": "^0.3.1",
    "@angular/animations": "^5.2.8",
    "@angular/common": "^5.2.8",
    "@angular/compiler": "^5.2.8",
    "@angular/core": "^5.2.8",
    "@angular/forms": "^5.2.8",
    "@angular/http": "^5.2.8",
    "@angular/platform-browser": "^5.2.8",
    "@angular/platform-browser-dynamic": "^5.2.8",
    "@angular/router": "^5.2.8",
    "angular-2-dropdown-multiselect": "^1.6.0",
    "angular-input-masks": "^4.1.0",
    "angular-toastr": "^2.1.1",
    "angular2-multiselect-dropdown": "^1.3.0",
    "angular2-text-mask": "^8.0.4",
    "angular2-toaster": "^4.0.1",
    "angularx-social-login": "^1.1.7",
    "aws-sdk": "^2.180.0",
    "bootstrap": "^4.0.0-beta.3",
    "brmasker4": "^1.0.1",
    "core-js": "^2.4.1",
    "crypto-js": "^3.1.9-1",
    "moment": "^2.20.1",
    "ng-block-ui": "^1.0.0-beta.18",
    "ng2-bootstrap": "^1.6.3",
    "ng2-checklist": "^1.0.2",
    "ng2-select": "^2.0.0",
    "ng2-semantic-ui": "^0.9.6",
    "ngx-auth": "^2.2.0",
    "ngx-bootstrap": "^2.0.2",
    "rxjs": "^5.5.2",
    "zone.js": "^0.8.14"
  },
  "devDependencies": {
    "@angular/cli": "1.6.3",
    "@angular/compiler-cli": "^5.2.8",
    "@angular/language-service": "^5.2.8",
    "@types/jasmine": "~2.5.53",
    "@types/jasminewd2": "~2.0.2",
    "@types/node": "~6.0.60",
    "codelyzer": "^4.0.1",
    "jasmine-core": "~2.6.2",
    "jasmine-spec-reporter": "~4.1.0",
    "karma": "~1.7.0",
    "karma-chrome-launcher": "~2.1.1",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "^1.2.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.1.2",
    "ts-node": "~3.2.0",
    "tslint": "~5.7.0",
    "typescript": "~2.4.2"
  }
}
```
What am I doing wrong?

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "element-closest": "^2.0.2",
     "extend": "^3.0.1",
     "is-ua-webview": "^1.0.0",
-    "popper.js": "^1.11.1",
+    "popper.js": "^1.14.0",
     "rxjs": "^5.0.1"
   },
   "devDependencies": {

--- a/src/misc/util/services/positioning.service.ts
+++ b/src/misc/util/services/positioning.service.ts
@@ -1,13 +1,13 @@
 import { ElementRef } from "@angular/core";
-import Popper from "popper.js";
+import Popper, { Modifiers, PopperOptions, Placement, Data } from "popper.js";
 
-type PopperModifiers = Popper.Modifiers & {
+type PopperModifiers = Modifiers & {
     computeStyle?:{
         gpuAcceleration:boolean;
     };
 };
 type PopperInstance = Popper & {
-    options:Popper.PopperOptions & {
+    options:PopperOptions & {
         modifiers:PopperModifiers;
     };
 };
@@ -43,7 +43,7 @@ export interface IPositionBoundingBox {
     right:number;
 }
 
-function placementToPopper(placement:PositioningPlacement):Popper.Placement {
+function placementToPopper(placement:PositioningPlacement):Placement {
     if (!placement || placement === PositioningPlacement.Auto) {
         return "auto";
     }
@@ -67,7 +67,7 @@ function placementToPopper(placement:PositioningPlacement):Popper.Placement {
     }
 
     // Join with hyphen to create Popper compatible placement.
-    return chosenPlacement.join("-") as Popper.Placement;
+    return chosenPlacement.join("-") as Placement;
 }
 
 function popperToPlacement(popper:string):PositioningPlacement {
@@ -112,7 +112,7 @@ export class PositioningService {
     public readonly subject:ElementRef;
 
     private _popper:PopperInstance;
-    private _popperState:Popper.Data;
+    private _popperState:Data;
     private _placement:PositioningPlacement;
 
     public get placement():PositioningPlacement {
@@ -133,7 +133,7 @@ export class PositioningService {
         return popperToPlacement(this._popperState.placement);
     }
 
-    public get state():Popper.Data {
+    public get state():Data {
         return this._popperState;
     }
 

